File: src/codes/Users_Manual/modulation_loop.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Modulation;
~using static AUTD3Sharp.Units;
~{
var m = new Sine(150u * Hz).WithLoopBehavior(LoopBehavior.Infinite); // infinite loop
~}
~{
var m = new Sine(150u * Hz).WithLoopBehavior(LoopBehavior.Finite(10u)); // 10 times
~}
~{
var m = new Sine(150u * Hz).WithLoopBehavior(LoopBehavior.Once); // equivalent to Finite(1u)
~}

File: src/codes/Users_Manual/modulation_loop.py
~from pyautd3 import LoopBehavior, Hz
~from pyautd3.modulation import Sine
m = Sine(150 * Hz).with_loop_behavior(LoopBehavior.Infinite)  # infinite loop
m = Sine(150 * Hz).with_loop_behavior(LoopBehavior.Finite(10))  # 10 times loop
m = Sine(150 * Hz).with_loop_behavior(LoopBehavior.Once)  # equivalent to finite(1)